disnake/appinfo.py

```diff
@@ -280,6 +288,95 @@ def summary(self) -> str:
         )
         return self._summary
 
+    async def edit(
+        self,
+        *,
+        description: Optional[str] = MISSING,
+        flags: ApplicationFlags = MISSING,
+        icon: Optional[AssetBytes] = MISSING,
+        cover_image: Optional[AssetBytes] = MISSING,
+        custom_install_url: Optional[str] = MISSING,
+        install_params: Optional[InstallParams] = MISSING,
+        role_connections_verification_url: Optional[str] = MISSING,
+        interactions_endpoint_url: Optional[str] = MISSING,
+        tags: Optional[List[str]] = MISSING,
```

Comment:
Figured this is worth mentioning - most of these fields, other than `icon` and `cover_image`, aren't marked as nullable in the documentation.
However considering that they *are* optional in the application object structure itself, clearly only setting them to `null` would reset the fields. So I assume this is fine?